debugtools/DDR_VM/generate.properties

```diff
@@ -44,6 +44,7 @@ linux_x86-64_combo,linux_x86-64_cmprssptrs_combo,linux_x86-64_cmprssptrs_panama
 ddr.order=aix_ppc-64,aix_ppc-64_cmprssptrs,aix_ppc,\
 linux_390-64,linux_390-64_cmprssptrs,linux_390,\
 linux_ppc-64,linux_ppc-64_cmprssptrs,linux_ppc,\
+linux_ppc-64_le,linux_ppc-64_cmprssptrs_le,\
```

Comment:
Please also add `linux_ppc-64_le_gcc` and `linux_ppc-64_cmprssptrs_le_gcc`.

Reply:
done